api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -222,6 +222,17 @@ message GrpcJsonTranscoder {
   // This is to support `HTML 2.0 <https://tools.ietf.org/html/rfc1866#section-8.2.1>`_
   bool query_param_unescape_plus = 12;
 
+  // If true, try to match the custom verb even if it is unregistered. By
+  // default, only match when it is registered.
+  //
+  // According to the http template `syntax <https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L226-L231>`_,
+  // the custom verb is `":" LITERAL` at the end of http template.
+  //
+  // For a request with /foo/bar:baz and ":baz" is not registered in any url_template, here is the behavior change
+  // * if the field is not set, ":baz" will not be treated as custom verb, so it will match /foo/{x=*}.
+  // * if the field is set, ":baz" is treated as custom verb,  so it will NOT match /foo{x=*} since the template doesn't use any custom verb.
```

Comment:
These two lines have different expressions, `/foo/{x=*}` vs `/foo{x=*}`. Is this deliberate?

Reply:
Good catch! Done.